be/src/vec/aggregate_functions/aggregate_function_collect.h

```diff
@@ -678,7 +678,7 @@ class AggregateFunctionCollect
                                             Arena* arena, const size_t num_rows) const override {
         if constexpr (ShowNull::value) {
             for (size_t i = 0; i != num_rows; ++i) {
-                if (places[i] + offset) {
+                if (places[i] || offset) {
```

Comment:
We should not change the logic while fix compile error. 

offset is added from pr https://github.com/apache/doris/pull/32387, please have a review @amorynan

Reply:
place[i] is a pointer, and offset is an unsigned int, so if placces[i] != null or offset != 0, equals places[i] + offset != null

Reply:
maybe the correct answer is like: `if(places[i] && offset < places[i].size())`, check `offset != 0 or == 0` is strange here.

Reply:
here below we use `places[i]+offset` as an pointer. so maybe `places[i] + offset != nullptr` is correct and more readable